src/docs/product/session-replay/index.mdx

```diff
@@ -4,23 +4,9 @@ sidebar_order: 3
 description: "Learn about Sentry's Session Replay, which provides video-like reproductions of user interactions on a site or web app."
 ---
 
-<Note>
 
-Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions or feedback, please email us at [support@sentry.io](mailto:support@sentry.io).
+<Include name="beta-note-session-replay.mdx" />
 
-**Currently, this feature is supported for only browser JavaScript, including these frameworks:**
```

Comment:
"only browser JavaScript, including these frameworks:" or 

"most all JavaScript major javascript frameworks:"

:D only sounds like it isn't so much, and yall did A LOT 👍